src/worker/predictive-text-studio-worker-impl.ts

```diff
@@ -21,6 +22,14 @@ export class PredictiveTextStudioWorkerImpl
     return await linkStorageToKmp(this.storage);
   }
 
+  async readGoogleSheet(
+    name: string,
+    wordListObject: WordList
+  ): Promise<ArrayBuffer> {
+    this.storage.saveFile(name, wordListObject);
+    return await linkStorageToKmp(this.storage);
```

Comment:
This behaviour will change when merged with `production`, but it's okay for this PR 😅